src/main/java/uk/gov/ons/census/casesvc/service/QuestionnaireLinkedService.java

```diff
@@ -1,5 +1,6 @@
 package uk.gov.ons.census.casesvc.service;
 
+import static net.logstash.logback.encoder.org.apache.commons.lang.StringUtils.isEmpty;
```

Comment:
it's probably exactly the same, functionality wise, but we have used the Spring version of this everywhere else I've seen it.

Reply:
Changed 👍